### DISCUSSION TITLE
400 - server_error

### DISCUSSION THREAD

**Discussion by MrMattn** [flobz/psa_car_controller#731] [General]
Hi
what can I do? Does anyone have a tip for me?

![Bildschirmfoto 2024-01-15 um 14 55 30](https://github.com/flobz/psa_car_controller/assets/50632820/dbd875a7-02bb-4286-8b3c-7ef57fb14db1)
![Bildschirmfoto 2024-01-15 um 14 57 02](https://github.com/flobz/psa_car_controller/assets/50632820/370054bb-98d4-4676-98c2-e5e3ea6a956b)
![Bildschirmfoto 2024-01-15 um 14 56 03](https://github.com/flobz/psa_car_controller/assets/50632820/df20aa44-bade-42c7-8d56-7dffdf94ed19)

**Comment by mitasa** [comment]
The MyOpel site has been down for almost a week. Today it has returned, but there has probably been some changes in the site because this problem persists.

**Reply by uzeller** [reply]
I just tried to log in to the MyOpel site and was notified that the domain had moved and was redirected. It all happened to fast for me to notice from where I was redirected. 
Maybe that information can help narrowing down the 400 - server error.

**Comment by ritschi86** [comment]
Same Issue my Opel Website is redirected to https://my.opel.com/de/de/vehicle-services

My Opel Site looks different

**Comment by atomizer83** [comment]
Hi, I'm running 3.3.1 and have the same issue.
I cannot register PSA car controller, but the real app, and login on https://my.opel.com works just fine.

The error and log are exactly the same as OP.

If anyone have any insight as to what could be causing this, feel free to comment.
I checked the fixes in 3.3.2, and unfortunately nothing seem to fix this particular error.

Thank you in advance, and thank you for the huge work you put into PSA Car Controller.

/Martin

**Reply by ShinZu3388** [reply]
Exactly the same behavior on my side.

**Comment by gerar19** [comment]
With Peugeot same here. Same error.

**Comment by ShinZu3388** [comment]
Same here, I was just setting up everything on my raspberry and get the identical error message... Login to my.opel.com works fine though.

**Comment by pauborrasblanco** [comment]
Same issues here. They must have been changing how the API works or something, to accomodate their new e-routes app. Guess we just need to wait.

**Comment by ict** [comment]
This is discussed in #733 in detail. As far as I understand the issue, no new registrations of psacc are possible at the moment, but existing installations still work. At least mine does *knocks on wood*.

**Reply by ShinZu3388** [reply]
Thank you. sad to see Stellantis is shutting out developers like psacc and evcc without providing an alternative way to use their API.

hopefully this will be solved soon.
